xrefs/network/fetch.json

```diff
@@ -200,7 +215,7 @@
     "responsetype": "responsetype",
     "same-origin data-url flag": "same-origin-data-url-flag",
     "sandboxed-storage-area-urls flag": "sandboxed-storage-area-urls-flag",
-    "should response to request be blocked due to nosniff?": "should-response-to-request-be-blocked-due-to-nosniff?",
```

Comment:
I don't know: Just checking out the fetch repository and running anolis make this change, Is my settings wrong?